hack/quick_install.sh

```diff
@@ -134,7 +134,7 @@ EOF
    echo "😀 Successfully installed Knative"
 fi
 
-if [ $installKserve = false ]; then
+if [ "${installKserve}" = false ]; then
```

Comment:
In bash, it should always check the string.
```suggestion
if [ "${installKserve}" == "false" ]; then
```

Reply:
same as above.